monkeytype/typing.py

```diff
@@ -251,8 +251,19 @@ def rewrite(self, typ):
         return typ
 
 
+class RewriteGenerator(TypeRewriter):
+    """Returns an Iterator, if the send_type and return_type of a Generator is None"""
+
+    def visit_Generator(self, typ):
+        args = getattr(typ, '__args__')
```

Comment:
this should probably just be `args = typ.__args__`, the getattr doesn't buy us anything